apps/core/components/sectorsList/SectorsList.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+
+import * as React from 'react';
+import { graphql, createFragmentContainer } from '@kiwicom/margarita-relay';
+
+import SectorDetail from './SectorDetail';
+
+type Props = {
+  +data: ?$ReadOnlyArray<?{|
+    +$fragmentRefs: any,
+  |}>,
+};
```

Comment:
This is a little bit weird, but I see that the generated flow types generates errors. We should check this against `@kiwicom/relay` package. Could you add some TODO comment so that we don't forget this.

Reply:
Right. TODO comment added.